### PR TITLE
add Dispatcher.

### PR DESCRIPTION
**Pull request by shy1st**
extract the "add Dispatcher" PR from the code in PR: #111。

### CODE REVIEW

File: src/main/java/org/casbin/jcasbin/persist/Dispatcher.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+package org.casbin.jcasbin.persist;
```
Comment (hsluoyz) [inline src/main/java/org/casbin/jcasbin/persist/Dispatcher.java:1]: plz add header:

```
// Copyright 2020 The casbin Authors. All Rights Reserved.
//	//
// Licensed under the Apache License, Version 2.0 (the "License");	// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.	// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at	// You may obtain a copy of the License at
//	//
//      http://www.apache.org/licenses/LICENSE-2.0	//      http://www.apache.org/licenses/LICENSE-2.0
//	//
// Unless required by applicable law or agreed to in writing, software	// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,	// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.	// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and	// See the License for the specific language governing permissions and
// limitations under the License.	// limitations under the License.
```

Comment (hsluoyz) [reply src/main/java/org/casbin/jcasbin/persist/Dispatcher.java:1]: @shy1st the format seems wrong. Please copy other header and change year to 2020

Comment (shy1st) [reply src/main/java/org/casbin/jcasbin/persist/Dispatcher.java:1]: @hsluoyz Thanks for your advice,I will deal with it.

**Comment by hsluoyz** [comment]
lgtm @shy1st next time please squash the update commits into the previous one.

**Comment by shy1st** [comment]
> lgtm @shy1st next time please squash the update commits into the previous one.

well,I will do that

**Pull request merged**